Update docs of StressChaos (#239)
chaos-mesh/website: docs/simulate-heavy-stress-on-kubernetes.md

| Parameter | Type   | Description                                              | Default value | Required | Example |
| --------- | ------ | -------------------------------------------------------- | ------------- | -------- | ------- |
| workers   | int    | Specifies the number of threads that apply memory stress |               | No      | `1`     |
| size      | string | Specifies the size of memory to be occupied or a percentage of the total memory size, and the final sum of allocated memory size is `workers * size`     |               | No       | `256MB`/`25%` |
| time      | string | Specifies the time to reach the memory `size`, and the growth model is a linear model     |               | No       | `10min` |

> (**tips**: Chaos Mesh uses [memStress](https://github.com/chaos-mesh/memStress) to simulate memory stressChaos Mesh uses memStress to simulate memory stress, in which way we can avoid high CPU load.)

Reviewer: "avoid high CPU load" from what? 

Please append descriptions for different behavior between memStress and stress-ng in here. Or put descriptions in the README of memStress, then put a link here.

Reviewer: How about using `:::note`? like 
```markdown
:::note
Chaos Mesh uses [memStress](https://github.com/chaos-mesh/memStress) to simulate memory stressChaos Mesh uses memStress to simulate memory stress, in which way we can avoid high CPU load.
:::

```

Author: I added some description of why we use memStress.

Author: PTAL @STRRL : )

##### CPUStressor

| Parameter | Type | Description | Default value | Required | Example |
| --- | --- | --- | --- | --- | --- |
| workers | int | Specifies the number of threads that apply CPU stress |  | Yes | `1` |

> | load | int | Specifies the percentage of CPU occupied 0 means sleep and no load; 100 means full load, and the final sum of CPU load is `workers * load` |  | No | `50` |

Reviewer: ```suggestion
| load | int | Specifies the percentage of CPU occupied. 0 means sleep and no load; 100 means full load, and the final sum of CPU load is `workers * load` |  | No | `50` |
```

| Parameter | Type   | Description                                              | Default value | Required | Example |
| --------- | ------ | -------------------------------------------------------- | ------------- | -------- | ------- |
| workers   | int    | Specifies the number of threads that apply memory stress |               | No      | `1`     |

> | size      | string | Specifies the size of memory to be occupied or a percentage of the total memory size, and the final sum of allocated memory size is `workers * size`     |               | No       | `256MB`/`25%` |

Reviewer 2: ```suggestion
| size      | string | Specifies the memory size to be occupied or a percentage of the total memory size. The final sum of the occupied memory size is `workers * size`.  |               | No       | `256MB`/`25%` |
```

> | time      | string | Specifies the time to reach the memory `size`, and the growth model is a linear model     |               | No       | `10min` |

Reviewer 2: ```suggestion
| time      | string | Specifies the time to reach the memory `size`. The growth model is a linear model.     |               | No       | `10min` |
```

> :::note
> Because [memStress](https://github.com/chaos-mesh/memStress) simulates memory stress by actually occupying memory rather than applying memory read and write stress, Chaos Mesh uses memStress to avoid the high CPU load caused by stress-ng applying read and write stress
> :::

Reviewer 2: ```suggestion
:::note

To avoid the high CPU load caused by the read and write pressure from `stress-ng`, Chaos Mesh uses [memStress](https://github.com/chaos-mesh/memStress) to simulate memory stress. This is because memStress simulates memory stress by consuming actual memory instead of applying the read and write pressure to memory.

:::
```

##### CPUStressor

| Parameter | Type | Description | Default value | Required | Example |
| --- | --- | --- | --- | --- | --- |
| workers | int | Specifies the number of threads that apply CPU stress |  | Yes | `1` |

> | load | int | Specifies the percentage of CPU occupied. 0 means sleep and no load; 100 means full load, and the final sum of CPU load is `workers * load` |  | No | `50` |

Reviewer 2: ```suggestion
| load | int | Specifies the percentage of CPU occupied. `0` means the CPU is in the sleep mode, and it has no load, while `100` refers to full load. The final sum of CPU load is `workers * load` |  | No | `50` |
```

Author: Actually, `0` means stressChaos doesn't add additional CPU load, not that CPU is in sleep mode.

Reviewer 2: > Actually, `0` means stressChaos doesn't add additional CPU load, not that CPU is in sleep mode.

have fixed, PTAL, thanks!

| --------- | ------ | -------------------------------------------------------- | ------------- | -------- | ------- |
| workers   | int    | Specifies the number of threads that apply memory stress |               | No      | `1`     |
| size      | string | Specifies the memory size to be occupied or a percentage of the total memory size. The final sum of the occupied memory size is `workers * size`.  |               | No       | `256MB / 25%` |
| time      | string | Specifies the time to reach the memory `size`. The growth model is a linear model.     |               | No       | `10min` |

:::note

To avoid the high CPU load caused by the read and write pressure from `stress-ng`, Chaos Mesh uses [memStress](https://github.com/chaos-mesh/memStress) to simulate memory stress. This is because memStress simulates memory stress by consuming actual memory instead of applying the read and write pressure to memory.

:::
##### CPUStressor

| Parameter | Type | Description | Default value | Required | Example |
| --- | --- | --- | --- | --- | --- |
| workers | int | Specifies the number of threads that apply CPU stress |  | Yes | `1` |

> | load | int | Specifies the percentage of CPU occupied. `0` means that no additional CPU is added, and it has no load, while `100` refers to full load. The final sum of CPU load is `workers * load`. |  | No | `50` |

Reviewer 2: ```suggestion
| load | int | Specifies the percentage of CPU occupied. `0` means that no additional CPU is added, and the CPU has no load. `100` refers to full load. The final sum of CPU load is `workers * load`. |  | No | `50` |
```
